docs/subnets/reference-elastic-subnets-parameters.md

```diff
@@ -0,0 +1,224 @@
+# Elastic Subnets Parameters
+
+Avalanche Permissioned Subnets turn into Elastic Subnet via the `TransformSubnetTx` transaction.
+`TransformSubnetTx` specifies a set of structural parameters for the Elastic Subnet.
+This reference describes these structural parameters and illustrates the constraints they must satisfy.
+
+## Elastic Subnet Parameters
+
+### `Subnet`
+
+`Subnet` has type `ids.ID` and it's the Subnet ID.
+`Subnet` is the ID of the `CreateSubnetTx` transaction that created the Subnet in the first place. 
+The following constraints apply:
+
+* `Subnet` must be different from `PrimaryNetworkID`.
+
+### `AssetID`
+
+`AssetID` has type `ids.ID` and it's the ID of the asset to use when staking on the Subnet.
+The following constraints apply:
+  
+* `AssetID` must not be the `Empty ID`.
+* `AssetID` must not be `AVAX ID`.
```

Comment:
Should we use `AVAX Asset ID` ?

Reply:
Reworded slightly but I trust your native speaker ear more than mine